warehouse/packaging/models.py

```diff
@@ -449,31 +467,20 @@ def __table_args__(cls):  # noqa
         ForeignKey("release_descriptions.id", onupdate="CASCADE", ondelete="CASCADE"),
         index=True,
     )
-    description: Mapped[Description] = orm.relationship(
-        backref=orm.backref(
```

Comment:
Sure thing!

This commit is specifically the one that addresses this change: [`50f6c96` (#14788)](https://github.com/pypi/warehouse/pull/14788/commits/50f6c9637d64276b110502e99a9a16fd27e08229)

While working through these, I found that the only place we created/removed a Description from a Release was in the specific test case, and nowhere else in the runtime code.

Instead of continuing to implement a relationship that isn't used, I opted to remove the "creation" characteristics.

HOWEVER! I think you found a bug!

Since you pointed it out, I dug a bit further into this relationship.

```sql
warehouse=> select count(*) from releases ;
  count
---------
 5000937
(1 row)

warehouse=> select count(*) from release_descriptions ;
  count
---------
 5620724
(1 row)

warehouse=> select count(d.id) from release_descriptions d
left join releases r on d.id = r.description_id
where r.description_id is null;
 count
--------
 619782
(1 row)

```

Since we expect descriptions to be a one-to-one relationship to a release, there's something amiss with how these 600k descriptions are orphaned, and likely due to the relationship definition.

I'll continue poking at this and get a change to this PR to actually delete a description when the release is deleted, and then we'll have to do a cleanup once it's correct.

Reply:
I've added [9947414](https://github.com/pypi/warehouse/pull/14788/commits/99474141382952521a69577843fd6386430fc602) to confirm the desired behavior, and the added only the necessary keys to ensure the behavior works.

Our dev db doesn't have this condition, as evidenced by 0 records returned for the same query, so I don't think we'll need a data migration, rather can execute the right query to clear out the orphan records. I'll do that only after this change has made it to prod, so we don't create more.

Reply:
Now deployed, the count from that query is 621360
If that remains the same next week, my plan is to remove these orphaned entries.

Reply:
The count is currently 621641 - which is higher than last, but not much higher. So I'm wondering what else is causing this before I perform any deletions.

Reply:
Noty gonna solve now, filed https://github.com/pypi/warehouse/issues/14825 to track it.